gtfs/spec/en/reference.md

```diff
@@ -411,6 +437,8 @@ To process the cost of a leg:
 | `network_id` | Foreign ID referencing `routes.network_id` | Optional | Identifies a route network that applies for the fare leg rule.<br><br>If there are no matching `fare_leg_rules.network_id` values to the `network_id` being filtered, empty `fare_leg_rules.network_id` will be matched by default.<br><br> An empty entry in `fare_leg_rules.network_id` corresponds to all networks defined in `routes.txt` excluding the ones listed under `fare_leg_rules.network_id` |
 | `from_area_id` | Foreign ID referencing `areas.area_id` | Optional | Identifies a departure area.<br><br>If there are no matching `fare_leg_rules.from_area_id` values to the `area_id` being filtered, empty `fare_leg_rules.from_area_id` will be matched by default. <br><br>An empty entry in `fare_leg_rules.from_area_id` corresponds to all areas defined in `areas.area_id` excluding the ones listed under `fare_leg_rules.from_area_id` |
 | `to_area_id` | Foreign ID referencing `areas.area_id` | Optional | Identifies an arrival area.<br><br>If there are no matching `fare_leg_rules.to_area_id` values to the `area_id` being filtered, empty `fare_leg_rules.to_area_id` will be matched by default.<br><br> An empty entry in `fare_leg_rules.to_area_id` corresponds to all areas defined in `areas.area_id` excluding the ones listed under `fare_leg_rules.to_area_id` |
+|  `from_timeframe_group_id` | Foreign ID referencing `timeframes.timeframe_group_id` | Optional |  Defines the timeframe for the fare validation event at the start of the fare leg.<br><br>The “start time” of the fare leg is the time at which the event is scheduled to occur.  For example, the time could be the scheduled departure time of a bus at the start of a fare leg where the rider boards and validates their fare. For the rule matching semantics below, the start time is computed in local time, as determined by [Local Time Semantics](#localtimesemantics) of [timeframes.txt](#timeframestxt).  The stop or station of the fare leg’s departure event should be used for timezone resolution, where appropriate.<br><br>For a fare leg rule that specifies a `from_timeframe_group_id`, that rule will match a particular leg if either `from_timeframe_group_id` is empty, or if there exists at least one row in `timeframes.txt` where all of the following conditions are true<br>- The row’s `timeframe_group_id` is equal to the `from_timeframe_group_id` value.<br>- The set of days identified by the row’s `service_id` contains the “current day” of the fare leg’s start time.<br>- The fare leg’s “time-of-day” is greater than or equal to the row’s `start_time` value and less than the `end_time` value.<br><br>An empty `fare_leg_rules.from_timeframe_group_id` indicates that the start time of the leg does not affect the fare. |
```

Comment:
> For example, the time could be the scheduled departure time of a bus at the start of a fare leg where the rider boards and validates their fare.

'Could be' makes little sense to me - are we just saying that data producers can invent what criteria apply to "start time" of a fare leg? This should be defined exhaustively, and I can't think of any cases other than the one mentioned here.

Reply:
Keeping as-is for now. Please note that we are working on adding data examples for time-variable fares, with the intention of showing how to represent the most common use-cases. (cc @Sergiodero)
Resolving this conversation.